File: Java/lab21_Strategy/task_2_2/src/Calculator.java
public class Calculator {
    public int add(int a, int b){
        int res = a + b;
        System.out.printf("%s + %s = %s%n", a, b, res);
        return res;
    }
    public int subtract(int a, int b){
        int res = a - b;
        System.out.printf("%s - %s = %s%n", a, b, res);
        return res;
    }
    public int multiply(int a, int b){
        int res = a * b;
        System.out.printf("%s * %s = %s%n", a, b, res);
        return res;
    }
}

File: Cpp/lab21_Strategy/task_2_2/src/calculator.cpp
#include <iostream>

#include "calculator.hpp"


int Calculator::add(int a, int b)
{
  int res = a + b;
  std::cout << a << " + " << b << " = " << res << std::endl;
  return res;
}

int Calculator::subtract(int a, int b)
{
  int res = a - b;
  std::cout << a << " - " << b << " = " << res << std::endl;
  return res;
}

int Calculator::multiply(int a, int b)
{
  int res = a * b;
  std::cout << a << " * " << b << " = " << res << std::endl;
  return res;
}
